modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsWorkloadByStep.testcase

```diff
@@ -246,43 +246,49 @@ definition {
 		property test.liferay.virtual.instance = "false";
 		property test.run.type = "single";
 
-		ApplicationsMenu.gotoPortlet(
-			category = "Workflow",
-			panel = "Applications",
-			portlet = "Process Builder");
+		task ("Given a workflow with parallel branches") {
```

Comment:
@thaiscabrals You can split setup into `Given.. and Given.. and Given` if needed